docs/tutorials/data-browser.md

```diff
@@ -0,0 +1,37 @@
+# Data Browser
+
+Marin comes with a data browser that makes it easy to
+view datasets (in various formats) and experiments produced by the executor.
+It is basically a file browser that handles popular file formats like jsonl and parquet.
+
+## Prerequisites
+
+- Basic [installation](installation.md)
+- Run an experiment, either [First Experiment](first-experiment.md) or [Executor 101](executor-101.md).
+
+## Using the data browser
+
+Install dependencies:
+
+```bash
+cd data_browser
+poetry install
+```
+
+The data browser takes a configuration file that specifies the root directory (in our examples, it's been `local_store`, 
+or `../local_store` if we're in the `data_browser` directory).  This is what's defined in `conf/local.conf`.
+
+The data browser has two pieces:
+- Frontend server (React)
+- Backend server (Flask)
+
+To start the data browser, we need to run both servers, in different shells:
+```bash
+DEV=true python server.py --config conf/local.conf
```

Comment:
Had to update this to make it work; @epicfaace has a `run-dev.py` script but that didn't work for me.  Probably eventually we should just have one way of doing it